src/api.js

```diff
@@ -23,11 +24,23 @@ export const getConferenceById = async id => {
 export const createNewConference = async conference => {
   try {
     const response = await axios.post(
-      "http://localhost:8080/conferences",
-      conference
+      "http://localhost:8080/conferences",conference);
+    return response.status;
+  } catch (error) {
+    console.log(error);
+    throw error;
+  }
+};
+
+export const deleteConferenceById = async id => {
+  try {
+    const response = await axios.delete(
+      `http://localhost:8080/conferences/${id}`
     );
+    toast.success("Conference successfully deleted!", { pauseOnHover: true });
```

Comment:
I don't think the `pauseOnHover` is needed as it is inherited from the container